### DISCUSSION TITLE
How to download simatic ax software?

### DISCUSSION THREAD

**Discussion by Owen-wei-767** [simatic-ax/.discussions#24] [General]
I didn't find a link that can be download simatic ax software. Pleas give me a link. thanks

**Comment by BeckerStS** [comment]
Hi @Owen-wei-767 

We are sorry ... but SIMATIC AX is not yet publicly available as of now. In addition we are in a very limited sales release and available in selected countries only.  For now please get in contact with your regional SIEMENS representative if you are interested and follow SIEMENS news for when SIMATIC AX will have its broad-release.  

We are looking forward to welcome you here to our simatic-ax community as well. 
Give us a follow if you like what you saw/ heard about us. 👍 

Greetings from the Siemens SIMATIC AX headquarter.
